src/cloud/utils.js

```diff
@@ -36,18 +36,31 @@ export function getServiceFilePath(environment, filename) {
 }
 
 export function readServiceYaml(environment, filename) {
-  const filePath = exports.getServiceFilePath(environment, filename);
-  return yaml.safeLoad(fs.readFileSync(filePath, 'utf8'));
+  const filePath = getServiceFilePath(environment, filename);
+  const content = fs.readFileSync(filePath, 'utf8');
+  const data = content.split(/^---$/gm).map((str) => {
+    return yaml.load(str);
+  });
+  if (data.length === 1) {
+    return data[0];
+  } else {
+    return data;
+  }
 }
 
-export function writeServiceYaml(environment, filename, data) {
-  const filePath = exports.getServiceFilePath(environment, filename);
-  const options = {
-    quotingType: '"',
-    forceQuotes: true,
-  };
-  const yamlString = yaml.safeDump(data, options);
-  return fs.writeFileSync(filePath, yamlString, 'utf8');
+export function writeServiceYaml(environment, filename, docs) {
+  if (!Array.isArray(docs)) {
+    docs = [docs];
+  }
+  const output = docs
+    .map((doc) => {
+      return yaml.dump(doc, {
```

Comment:
@andrewplummer Why dump instead of safeDump? Does this help to preserve double quotes in the yml files?

Reply:
actually I just bumped the js-yaml version which now only has `.dump()` which is safe by default